style/app.scss

```diff
@@ -2487,14 +2480,44 @@ main-filter input
 
 .tooltip-group-display
 {
-  cursor: pointer;
-  transition: opacity 0.4s linear;
-  opacity:1;
+  cursor     : pointer;
+  transition : opacity 0.4s linear;
+  opacity    : 1;
 }
 
 .tooltip-group-hidden
 {
-  cursor: pointer;
-  transition: opacity 0.4s linear;
-  opacity:0;
+  cursor     : pointer;
+  transition : opacity 0.4s linear;
+  opacity    : 0;
+}
+
+.graph-dropdown-hidden
+{
+  opacity    : 0;
+  position   : absolute;
+  display    : inline-block;
+  height     : 0;
+  overflow   : hidden;
+  transition : opacity 0.4s linear;
```

Comment:
I don't like these transitions. You can keep the timeout in the JS code, but remove both this rule and the one for graph-dropdown-display. Use the display rule.